js/lib/scene.js

```diff
@@ -1,17 +1,16 @@
 define(['basic/entity', 'geo/v2', 'core/graphic', 'config/screen', 'config/config'],
 		function (Entity, V2, graphics, screen, config) {
-			function Scene() {
+		    function Scene() {
 				Entity.call(this, Zero(), new V2(screen.w, screen.h));
 				this.keyAware = [];
+			    this.bgColor = "#DDD";
```

Comment:
I don't care which color is set by default as long as a random or whatever color is set